Source/WebCore/platform/ios/PlatformScreenIOS.mm

```diff
@@ -39,10 +38,13 @@
 #import "ScreenProperties.h"
 #import "WAKWindow.h"
 #import "Widget.h"
-#import <pal/cocoa/MediaToolboxSoftLink.h>
-#import <pal/ios/UIKitSoftLink.h>
+
 #import <pal/spi/ios/MobileGestaltSPI.h>
 #import <pal/spi/ios/UIKitSPI.h>
+#import <pal/system/ios/Device.h>
+
+#import <pal/cocoa/MediaToolboxSoftLink.h>
```

Comment:
The old layout was more correct (smooshed against the bottom of the quoted includes, in the same paragraph, sorted as `sort` would.

Reply:
(except for the soft links!)

Reply:
The bottom is a SoftLink, so I'm not sure what you're saying here.